snapcraft/commands/upload.py

```diff
@@ -54,13 +54,13 @@ def main(argv=None):
 
     # make sure the full lifecycle is executed
     yaml_config = snapcraft.yaml.load_config()
-    snap_name = _format_snap_name(yaml_config.data)
+    snap_package = _format_snap_name(yaml_config.data)
```

Comment:
Yeah I'd use snap but it clashes with the import, so I went with `snap_filename`.